src/System.Device.Gpio/System/Device/Spi/SpiDevice.cs

```diff
@@ -56,6 +66,7 @@ public abstract partial class SpiDevice : IDisposable
         /// </summary>
         /// <param name="settings">The connection settings of a device on a SPI bus.</param>
         /// <returns>A communications channel to a device on a SPI bus running on Windows 10 IoT.</returns>
+        [Obsolete]
```

Comment:
I honestly don't agree with this change. I think that most people using this will still probably not care about the Board abstraction, and what this is doing is probably forcing consumers to use it. Given this Create method is the main entrypoint, I would keep it without Obsolete and simply just add the other option of doing it via a Board.